modules/core/04-channel/keeper/upgrade.go

```diff
@@ -580,35 +591,35 @@ func (k Keeper) ChanUpgradeCancel(ctx sdk.Context, portID, channelID string, err
 		return errorsmod.Wrap(err, "failed to verify counterparty error receipt")
 	}
 
-	// If counterparty sequence is less than the current sequence, abort the transaction since this error receipt is from a previous upgrade.
-	// Otherwise, set our upgrade sequence to the counterparty's error sequence + 1 so that both sides start with a fresh sequence.
-	currentSequence := channel.UpgradeSequence
-	counterpartySequence := errorReceipt.Sequence
-	if counterpartySequence < currentSequence {
-		return errorsmod.Wrapf(types.ErrInvalidUpgradeSequence, "error receipt sequence (%d) must be greater than or equal to current sequence (%d)", counterpartySequence, currentSequence)
-	}
-
 	return nil
 }
 
+// isAuthorizedUpgrader checks if the sender is authorized to cancel the upgrade.
+func (Keeper) isAuthorizedUpgrader(signer string) bool {
```

Comment:
do you remember if we have an issue for this function? A quick glance at issue list didn't yield anything

Reply:
aye good call, we need to open one still https://github.com/cosmos/ibc-go/pull/4404#discussion_r1300125982

Reply:
https://github.com/cosmos/ibc-go/issues/4420 done